src/components/ChartTitle.tsx

```diff
@@ -68,9 +68,14 @@ const ChartTitle = (props: TChartTitleProps) => {
                     store={categoricalDisplay}
                     isNestedList={isNestedList}
                     searchInputClassName={searchInputClassName}
-                    onSelectItem={(x: TProcessedSymbolItem) => {
-                        if (x.symbol !== currentSymbol.symbol) {
-                            onChange(x.symbol);
+                    onSelectItem={(symbol_object: TProcessedSymbolItem, category_id: string) => {
+                        if (symbol_object.symbol !== currentSymbol.symbol) {
+                            onChange(symbol_object.symbol);
+                            state.stateChange(STATE.SYMBOL_CHANGE, {
+                                tab_market_name:
+                                    category_id === 'favorite' ? 'favorites' : getSymbolMarketCategory(symbol_object),
```

Comment:
Referring to this it could also be `symbol_market_category` name instead of `tab_market_name` https://github.com/binary-com/deriv-app/pull/9848/files#r1316656672